### PR TITLE
LPS-153062 Page/Layout overwrite warning when publishing

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/LPS-153062

@stsquared99 Still LGTM after testing your changes

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -16447,6 +16447,7 @@ this-product-does-not-have-any-released-versions=This product does not have any
 this-product-is-discontinued.-you-can-see-the-replacement-product-by-clicking-on-the-button-below=This product is discontinued. You can see the replacement product by clicking on the button below.
 this-product-was-updated=This product was updated.
 this-product-will-be-automatically-updated-to-a-newer-version=This product will be automatically updated to a new version.
+this-publication-contains-changes-to-a-content-page-publishing-these-changes-will-fully-overwrite-the-page's-content-and-layout-in-production=This publication contains changes to a content page. Publishing these changes will fully overwrite the page's content and layout in production.
```
Comment (stsquared99) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:16450]: Can you make sure there is a "." in the key? There should be a ton of examples in the file. Thanks!

**Review by stsquared99 — COMMENTED** [review]
Just the language key. Thanks!

**Comment by ctang3** [comment]
Updated, thanks Sam

**Comment by stsquared99** [comment]
ci:test:publications

**Comment by stsquared99** [comment]
ci:test:relevant

**Comment by stsquared99** [comment]
ci:test:sf

**Comment by stsquared99** [comment]
@tinatian Can you take a look? Thanks

**Comment by tinatian** [comment]
@xbrianlee Please help me scan test results.

Thanks
Tina.

**Comment by david-truong** [comment]
@suilin can you verify the ticket since Brian is on PTO

**Comment by suilin** [comment]
Hi @tinatian, the results look good, the failures are not related to the changes, thanks.

**Comment by tinatian** [comment]
ci:forward

**Pull request closed without merging**